backend/lib/endpoints/posts.js

```diff
@@ -175,7 +177,11 @@ async function routes(app) {
             { $match: { $and: filters, $text: { $search: keywords } } },
             { $sort: { score: { $meta: "textScore" } } },
           ]
-          : [{ $match: { $and: filters } }, { $sort: { _id: -1 } }];
+          : sort
+            ? [
+              { $sort: { [sort]: order === "asc" ? 1 : -1 } }
```

Comment:
it seems when the `sort` is passed in, the `$match: { $and: filters }` is removed, not sure if that's from the requirement or still need that ❔